packages/components/src/components/contract-card/contract-card.scss

```diff
@@ -709,8 +709,7 @@
     }
     &__text {
         position: absolute;
-        color: var(--status-success);
+        color: var(--status-success) !important;
```

Comment:
Can `!important` be avoided?  Please, ignore if not

Reply:
Cannot. Something changed with the build and Text component css is getting applied instead